common/schema-model/src/main/java/io/smallrye/graphql/schema/model/Scalars.java

```diff
@@ -181,6 +188,12 @@ public static void addUuid() {
         // Void
         populateScalar(Void.class.getName(), VOID, Void.class.getName());
         populateScalar(void.class.getName(), VOID, Void.class.getName());
+
+        if (Boolean.getBoolean("smallrye.graphql.federation.enabled")) {
```

Comment:
I wasn't sure if we can use `Config.get().isFederationEnabled()` here.

Reply:
Correct, MP Config is not available here, this module executes during Quarkus build time.